Alignment/HIPAlignmentAlgorithm/plots/HIPplots.cc

```diff
@@ -905,349 +915,349 @@ void HIPplots::plotAlignableChiSquare(char *plotName, float minChi2n){
-      //      hiter[i]->Draw("");
+      //hiter[i]->Draw("");
     }
-    else if((i+1)%5==0){
+    else if ((i+1)%5==0){
       hiter[i]->SetLineColor(colors[taken]);
       taken++;
-      sprintf(legend_entry,"Norm #chi^{2}; Iter %d; %.4f",i+1,histmean);
-      l->AddEntry(hiter[i],legend_entry,"L");
+      sprintf(legend_entry, "Norm #chi^{2}; Iter %d; %.4f", i+1, histmean);
+      l->AddEntry(hiter[i], legend_entry, "L");
       tmpmax=hiter[i]->GetBinContent(hiter[i]->GetMaximumBin());
-      if(tmpmax>newmax)newmax=tmpmax;
-      //      hiter[i]->Draw("same");
+      if (tmpmax>newmax)newmax=tmpmax;
+      //hiter[i]->Draw("same");
     }
   }
-  cout<<"NewMax after 1st loop -> "<<newmax<<endl;
+  cout << "NewMax after 1st loop -> " << newmax << endl;
 
-  for(i=0;i<maxIteration;i++){
+  for (i=0; i<maxIteration; i++){
     hiter[i]->SetMaximum(newmax*1.1);
-   if(i==1)      hiter[i]->Draw("");
-   else if((i+1)%5==0) hiter[i]->Draw("same");
+    if (i==1)      hiter[i]->Draw("");
+    else if ((i+1)%5==0) hiter[i]->Draw("same");
   }
   l->Draw();
 
-  sprintf(finplotname,"%s_distr.png",plotName);
-  cout<<finplotname<<endl;
+  sprintf(finplotname, "%s_distr.png", plotName);
+  cout << finplotname << endl;
   c_chi2ndist->SaveAs(finplotname);
   c_chi2ndist->SetLogy();
-  sprintf(finplotname,"%s_distrlog.png",plotName);
-  cout<<finplotname<<endl;
+  sprintf(finplotname, "%s_distrlog.png", plotName);
+  cout << finplotname << endl;
   c_chi2ndist->SaveAs(finplotname);
-  
+
   delete  c_chi2ndist;
   delete f;
 }//end plotAlignableChiSquare
 
 
 
-// -----------------------------------------------------------------
-// private classes
-int HIPplots::GetNIterations(TDirectory *f,char *tag,int startingcounter){
-	int fin_iter=0,i=startingcounter;
-	bool obj_exist=kTRUE;
-	while(obj_exist){
-		char objname[32];
-		sprintf(objname,"%s%d",tag,i);
-		if(!f->FindObjectAny(objname))obj_exist=kFALSE;
-		fin_iter=i;
-		i++;
-	}
-	cout<<"Max Iterations is "<<fin_iter<<endl;
-	
-	return fin_iter;
+//-----------------------------------------------------------------
+//private classes
+int HIPplots::GetNIterations(TDirectory* f, char* tag, int startingcounter){
+  int fin_iter=0, i=startingcounter;
+  bool obj_exist=kTRUE;
+  while (obj_exist){
+    char objname[32];
+    sprintf(objname, "%s%d", tag, i);
+    if (!f->FindObjectAny(objname))obj_exist=kFALSE;
+    fin_iter=i;
+    i++;
+  }
+  cout << "Max Iterations is " << fin_iter << endl;
+
+  return fin_iter;
 }
 
-int HIPplots::GetSubDet( unsigned int id ){
-	
-	const int reserved_subdetectorstartbit=25;
-	const int reserved_subdetectorfinalbit=27;
-	
-	unsigned int detID = id;
-	
-	int shift = 31-reserved_subdetectorfinalbit;
-	detID = detID<<(shift);
-	shift = reserved_subdetectorstartbit + shift;
-	detID = detID>>(shift);
-	
-	return detID;
-	
+int HIPplots::GetSubDet(unsigned int id){
+
+  const int reserved_subdetectorstartbit=25;
+  const int reserved_subdetectorfinalbit=27;
```

Comment:
No, they are not. The HIPplots.cc are pretty old by themselves. I am not sure if they will work with Phase-I geometry, but the purpose of this PR is not Phase-I; it is rather merging previous Phase-0 developments to be able to continue toward Phase-I.